utils/findLinks.js

```diff
@@ -0,0 +1,64 @@
+
+const chalk = require('chalk')
+
+
+const findLinks = (datos, urlF) => {
+    let allthelink = /\[(.*)\]\S((http|https):\/\/[^\s\n)]+)(?=\))/gim;
+    let regularL = /(http|https):\/\/.*\)?/gi;
+    let textReg = /\[(.*)\]/gim
+    let objectLinks = [];
+    let newFile = process.argv.indexOf('--file');
+    let uri = process.argv[newFile + 1];
+    let joinArray = []
+    let linkAndtext;
+    let joinObject;
+    return new Promise((resolve) => {
+        let noStr = datos;
```

Comment:
En ingles se veria mejor
```suggestion
        let noStr = data;
```